terria/africa.json

```diff
@@ -125,6 +125,18 @@
                      "type": "group",
                      "preserveOrder": true,
                      "items": [
+                        {
+                           "name": "Annual GeoMAD (Sentinel-2)",
+                           "type": "wms",
+                           "layers": "gm_s2_annual",
+                           "url": "https://ows-af.digitalearth.africa/",
```

Comment:
dont use `ows-af`,, use `ows` end

Reply:
@pindge is gm_s2_annual available from ows.digitalearth.africa?